tests/python/pytorch/sparse/test_elementwise_op.py

```diff
@@ -225,18 +233,36 @@ def test_sub_sparse_diag(val_shape):
     assert torch.allclose(dense_diff, -diff4)
 
 
-@pytest.mark.parametrize("op", ["mul", "truediv", "pow"])
```

Comment:
Look like the new test omits the case for "truediv" and "pow". Have they been covered in other test cases?